client/scripts/views/pages/discussions/index.ts

```diff
@@ -63,23 +64,38 @@ const getLastSeenDivider = (hasText = true) => {
   );
 };
 
-export const CommunityOptionsPopover: m.Component<
-  { isAdmin: boolean; isMod: boolean },
-  {}
-> = {
+export const CommunityOptionsPopover: m.Component<{}> = {
   view: (vnode) => {
-    const { isAdmin, isMod } = vnode.attrs;
+    const isAdmin =
+      app.user.isSiteAdmin ||
+      app.user.isAdminOfEntity({
+        chain: app.activeChainId(),
+        community: app.activeCommunityId(),
+      });
+    const isMod = app.user.isRoleOfCommunity({
+      role: 'moderator',
+      chain: app.activeChainId(),
+      community: app.activeCommunityId(),
+    });
     if (!isAdmin && !isMod && !app.community?.meta.invitesEnabled) return;
+
+    // add extra width to compensate for an icon that isn't centered inside its boundaries
+    const DropdownIcon = m('.dropdown-wrapper',
+    [
+      m(Icon, {
+        name: Icons.CHEVRON_DOWN,
+      }),
+      m('.dropdown-spacer', {})
```

Comment:
is there a reason we can't just add the spacing style directly to the Icon itself by adding a class to the Icon component?

like:
```
m(Icon, {
   name: Icons.CHEVRON_DOWN,
   class: '.dropdown-spacing',
}), // Then we can remove the wrapper and spacer? 
```

Reply:
I'm not totally sure why but the Icons that are being used all have a ::before pseudo-element in them that's causing a small extra space in the div, which makes the actual SVG offset from the div center. I wasn't able to override the behavior of the ::before with css in the way you described, so I added the extra div to essentially recenter the svg. Played around with this for a while and wasn't able to find a fix, if you think it's important I can keep looking, your call.

Reply:
@gdjohnson do you have any ideas or is this hack fine in your mind?

Reply:
It seems fine for now, I can poke into the ::before issue more generally on Monday, but until then this'll do.